isteven-multi-select.js

```diff
@@ -691,12 +691,21 @@ angular
                 if (tempMaxLabels > 0) {
                   $scope.varButtonLabel += ', ... ';
                 }
-                $scope.varButtonLabel +=
-                  'Showing ' +
-                  $scope.outputModel.length +
-                  ' ' +
-                  (attrs.itemNames ? attrs.itemNames : 'metric') +
-                  ($scope.outputModel.length === 1 ? '' : 's');
+                if (attrs.itemNames) {
+                  $scope.varButtonLabel =
```

Comment:
confirming that you intend to do an assignment here, but concatenation in the `else`?

Reply:
both seem to work, but it was an oversight on my part. changed it back for consistency